lib/active_model/serializable_resource.rb

```diff
@@ -9,6 +9,7 @@ def initialize(resource, options = {})
       @resource = resource
       @adapter_opts, @serializer_opts =
         options.partition { |k, _| ADAPTER_OPTION_KEYS.include? k }.map { |h| Hash[h] }
+      notify_active_support
```

Comment:
Hi, I take a look at the code and we have a logger but actually we do not log anything, I am right or miss something?

Since Rails provides [instrumentation](http://api.rubyonrails.org/classes/ActiveSupport/Notifications.html) I think we can invert this logic of capturing log output, instead we can use the instrumentation to generate the events and subscribe to these events like in this test helper, this way we do not need to open and parse log files.

And as a bonus we can generate the logs using instrumentation with [`ActiveSupport::Notifications`](http://api.rubyonrails.org/classes/ActiveSupport/Notifications.html) and the [`ActiveSupport::LogSubscriber`](http://api.rubyonrails.org/classes/ActiveSupport/LogSubscriber.html). 
Active Job does [something like that using `ActiveSupport::Notifications`](https://github.com/rails/rails/blob/d2ef471b849b28c6962e4ed784fbec42aae9f9ec/activejob/lib/active_job/logging.rb#L19-L38) and [`ActiveSupport::LogSubscriber`](https://github.com/rails/rails/blob/d2ef471b849b28c6962e4ed784fbec42aae9f9ec/activejob/lib/active_job/logging.rb#L54-L119).

Some references about intrumentation are [this post](http://www.railstips.org/blog/archives/2013/01/23/an-instrumented-library-in-30-lines/) and in the [Crafting Rails Application](https://pragprog.com/book/jvrails2/crafting-rails-4-applications) have a chapter about it.

This makes sense to you?